app/components/shared/Slider.vue.ts

```diff
@@ -1,42 +1,31 @@
 import Vue from 'vue';
-import vueSlider from 'vue-slider-component';
+import VueSlider from 'vue-slider-component';
 import { throttle } from 'lodash-decorators';
 import { Component, Prop } from 'vue-property-decorator';
 
 @Component({
-  components: { vueSlider }
+  components: { VueSlider }
 })
 export default class SliderInput extends Vue {
-
-  @Prop()
-  value: number;
-
-  @Prop()
-  min: number;
-
-  @Prop()
-  max: number;
-
-  @Prop()
-  interval: number;
-
-  @Prop()
-  disabled: boolean;
-
-  @Prop()
-  tooltip: string;
-
-  @Prop()
-  valueBox: boolean;
-
-  @Prop()
-  dotSize: number;
-
-  @Prop()
-  sliderStyle: object;
-
-  @Prop()
-  usePercentages: boolean;
+  @Prop() value: number;
+  @Prop() min: number;
+  @Prop() max: number;
+  @Prop() interval: number;
+  @Prop() disabled: boolean;
+  @Prop() tooltip: string;
+  @Prop() valueBox: boolean;
+  @Prop() dotSize: number;
+  @Prop() sliderStyle: object;
+  @Prop() usePercentages: boolean;
+
+  $refs: { slider: any };
+
+  mounted() {
+    // Hack to prevent transitions from messing up slider width
+    setTimeout(() => {
+      if (this.$refs.slider) this.$refs.slider.refresh();
+    }, 500);
```

Comment:
don't see any clearTimeout, it's a possible memory leak here

Reply:
It's just a timeout so it only fires once.  Instead of adding a `clearTimeout` I just added the `if` statement to check if the slider is still there.  I don't think this should be a leak.

Reply:
Oh, my bad. Somehow I have seen a `setInterval` here